extensions/resource-deployment/package.nls.json

```diff
@@ -80,5 +80,12 @@
 	"sql-azure-database-server-display-name": "Database Server",
 	"azure-sqldb-agreement": "I accept {0}, {1} and {2}.",
 	"azure-sqldb-agreement-sqldb-eula": "Azure SQL DB License Terms",
-	"azure-sqldb-agreement-azdata-eula": "azdata License Terms"
+	"azure-sqldb-agreement-azdata-eula": "azdata License Terms",
+	"azure-sql-mi-display-name": "Azure SQL managed instance",
+	"azure-sql-mi-display-description": "Create an SQL instance, a fully managed database service. Best for migration and applicatiohns requiring the broadest SQL Server engine compatibility",
```

Comment:
`Create a SQL instance`

This whole beginning of the sentence should be looked at that though - `Create an SQL instance, a fully managed database service` doesn't really make sense to me.